### DISCUSSION TITLE
[Refactor] : 몇가지 이름 변경 제안

### DISCUSSION THREAD

**Discussion by goldentrash** [gdgoc-konkuk/platform-core#47] [Ideas]
### `Member`

현재 DB Scheme 중 `Member`에 `id`와 `member_id`를 함께 사용하고 있습니다. 개인적으로, `member_id`를 보고 학번을 떠올리기가 쉽지 않았습니다. 좀 더 명확한 이름으로 교체를 제안합니다. 

이 경우, `Participant.memberId`도 `Participant.studentId`의 형태로 변경되어 더 명확하게 설정할 수 있습니다. 
(기존에는 field type이 `Long`인지 `String`인지를 통해 `id`와 `memberId`를 구분해야 했습니다)

- [x] rename : `Member.memberId` => `Member.studentId`
- [x] rename : `MemberRepository.findByMemberId`  => `MemberRepository.findByStudentId`
- [x] rename : `ParticipantRepository.findByMemberIdAndAttendanceId` => `ParticipantRepository.findByStudentIdAndAttendanceId`

### `MemberRole`

GDSC에서 GDG on Campus로 개편되며 명칭에 변화가 예정되어 있었을 뿐만 아니라, 동아리 회원 권한과 시스템(서비스) 권한을 분리하여 생각하기 위해서 현재의 `Lead` - `Admin` - `Member`의 형태로 `MemberRole`을 구성하였던 것으로 기억합니다. 

하지만 현재 회원 명칭은 "크게 상관 없다"는 상황으로 보이며, 동아리 내부에서는 여전히 `Lead` - `Core` - `Member`의 명칭을 사용하고 있습니다. 또한, 시스템(서비스) 권한과 회원 권한을 동일하게 가져가는 상황에서 `Admin` 권한을 `Core`로 재설정하는게 어떨까 싶습니다.

- [x] reanme : `MemberRole.Admin` => `MemberRole.Core`

### `TaskNotFoundException`

`TaskNotFoundException`의 현 위치는 `global/exceptions`입니다. `global/scheduler`로의 이동을 제안합니다. `TaskNotFoundException`는 프로그램 전역으로 발생하는 예외라기보다는 스케쥴러를 사용할 때 발생하는 예외로 보이기 때문입니다.

- [x] move : `global/exceptions/TaskNotFoundException` => `global/scheduler/TaskNotFoundException`

**Comment by ekgns33** [comment]
흠좀무

**Reply by ekgns33** [reply]
진행시키시죠
